app/connectors/getFinancialDetails/FinancialDetailsHipConnector.scala

```diff
@@ -47,24 +47,28 @@ class FinancialDetailsHipConnector @Inject() (httpClient: HttpClient, appConfig:
   )
 
   def getFinancialDetails(enrolmentKey: AgnosticEnrolmentKey, includeClearedItems: Boolean)(implicit
-      hc: HeaderCarrier): Future[FinancialDetailsResponse] = {
```

Comment:
This file is just switching to the new HIPParser since I've split the IF and HIP